frontend/.eslintrc.js

```diff
@@ -38,6 +38,15 @@ module.exports = {
     "@typescript-eslint/camelcase": 0,
     // Disable prop-types as we use TypeScript for type checking
     "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        args: "after-used",
+        argsIgnorePattern: "^_",
+        ignoreRestSiblings: false,
+        vars: "all",
+      },
+    ],
```

Comment:
This explicit rule is just to tweak an already active rule to ignore the pattern, correct?

Might be worth adding a comment with a few examples captured by the pattern

Reply:
Great idea! Done 🔥